xrtpy/response/effective_area.py

```diff
@@ -470,24 +583,61 @@ def filter_contamination_transmission(self):
         return [abs(transmittance[i] ** 2) for i in range(4000)]
 
     @property
-    def interpolated_filter_contamination_transmission(self):
+    def _interpolated_filter_contamination_transmission(self):
         """Interpolate filter contam transmission to the wavelength."""
         Filter_contam_transmission = interpolate.interp1d(
-            self.n_DEHP_wavelength, self.filter_contamination_transmission
+            self.n_DEHP_wavelength, self._filter_contamination_transmission
         )
         return Filter_contam_transmission(self.channel_wavelength)
 
     @u.quantity_input
     def effective_area(self) -> u.cm**2:
-        """Calculation of the Effective Area."""
+        """
```

Comment:
```suggestion
        r"""
```
Because the docstring includes a `\` which is an escape character